lib/src/base-styles/_nx-colors.scss

```diff
@@ -196,6 +196,31 @@
   // NxTree
   --nx-color-tree-line: var(--nx-swatch-grey-50);
 
+  // NxStatusIndicator
+  --nx-color-status-indicator-negative-background: var(--nx-swatch-grey-95);
+  --nx-color-status-indicator-negative-border: var(--nx-swatch-grey-30);
+  --nx-color-status-indicator-negative-text: var(--nx-swatch-grey-10);
+  --nx-color-status-indicator-negative-icon-background: var(--nx-swatch-grey-60);
+  --nx-color-status-indicator-negative-icon-border: var(--nx-swatch-grey-20);
+
+  --nx-color-status-indicator-intermediate-background: var(--nx-swatch-blue-95);
+  --nx-color-status-indicator-intermediate-border: var(--nx-swatch-blue-30);
+  --nx-color-status-indicator-intermediate-text: var(--nx-swatch-blue-10);
+  --nx-color-status-indicator-intermediate-icon-background: var(--nx-swatch-blue-30);
```

Comment:
In sketch, the color of intermediate icon background is set to `nx-blue-60` in light mode

Reply:
> In sketch, the color of intermediate icon background is set to `nx-blue-60` in light mode

Good catch! I ran the light mode visual test and didn't see any new snapshots, so I thought I all of my color variables were correct. But there's actually a bug in `main` currently. If you go to gallery.sonatype.dev it actually has the wrong background color (it's blue-30). 😄 

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/1069/commits/300089e2f6d9572ae9faeb9766ee39f9e65c7b15

Reply:
Yes the color in main now is incorrect. This should fix that once merged.